decompressed/gui_file/www/lang/it-it/webui-core.po

```diff
@@ -2036,10 +2036,10 @@ msgid "Processing"
 msgstr "Elaborazione"
 
 msgid "Product Name"
-msgstr "Nome prodotto"
```

Comment:
Mi sembra giusto lasciare prodotto

Reply:
Nella pagina condivisione stampante "Nome prodotto" credo sia corretto mettere "Nome stampante",
Fosse un altar funzione (es. condovisioen disco) avrebbe senso lasciare nome proddotto ma non in condivisione stampante.

Reply:
sono daccordo il problema e' che la traduzione deve solo tradurre senza cambiare il senso delle frasi, se c'e scritto prodotto in inglese e' da tradurre in prodotto o in alternativa cambiare anche la stringa inglese in modo che ci sia scritto printer cio' per non alterare quello che poi e' il contenuto della stringa originale e rendere tutto coerente (e soprattutto per il motivo per cui molte stringhe vengono riutilizzate per cui magari e' volutamente immesso un nome piu' generico)